### PR TITLE
Recruitment: restart banners

### PR DESCRIPTION
**Pull request by breville**
### Homepage for Code Break

Screenshot 2020-05-12 10 55 08

### Homepage not for Code Break

Screenshot 2020-05-11 23 38 25

### https://studio.code.org/courses?view=teacher 

Screenshot 2020-05-12 10 05 10

### https://studio.code.org/home for signed-in teacher

Screenshot 2020-05-12 10 06 44

### https://studio.code.org/courses

Screenshot 2020-05-11 23 40 13

### https://code.org/yourschool

Screenshot 2020-05-12 09 57 39

### CODE REVIEW

**Comment by Erin007** [comment]
The call to action everywhere is "Join us", except on /yourschool where it's "Apply Now" - double checking that's intentional?

**Comment by Erin007** [comment]
I have nit picky wonders about the signed out homepage buttons and text update because it's kind of unclear what I should be checking out. The text says "Check out our other home learning resources" but the "Check it out" button refers to Code Break. Maybe "Take a Break" instead of "Check it out"? 🤷

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 1]
```diff
@@ -47,10 +47,18 @@
...
+            %img{src: "images/homepage/codebreak_check_button.png", style: "width: 250px; "}
+      - elsif entry[:type] == "code_break_home"
+        %div
+          %a{href: "/athome"}
+            %img{src: "images/homepage/codebreak_home_button.png", style: "width: 250px; margin-top: 10px"}
       - elsif entry[:type] == "code_break_video"
         %div
           %a{onclick: "return showVideo_mainvideo();", style: "cursor: pointer"}
-            %img{src: "images/homepage/watch_video_drawn_button.png", style: "width: 250px; margin-top: 10px; "}
+            %img{src: "images/homepage/watch_video_drawn_button.png", style: "width: 250px; margin-top: 10px"}
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml:61]: keeping this around in case we need it again even though it's not currently in use?

**Comment by Erin007** [comment]
Looks like test failures are legit, but should be straightforward to resolve.

**Review by Erin007 — APPROVED** [review]
A couple questions, but code changes look good to me! Approved pending test updates.

File: pegasus/sites.v3/code.org/views/homepage_hero.haml (hunk 1 above)
Comment (breville) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml:61]: For now, yes.  Also I forgot :)

**Comment by ericfershtman** [comment]
Handful of comments:

1. re Erin's comment about the Apply Now button on /yourschool: Let's change it to "Join us" so it's consistent with the other banners.

2. re Erin's comment about buttons on  Code Break version of the signed-out homepage: Good call, let's change it the top button from "Check it out" to "Take a Code Break" if that fits. 

3. The image for the callout box on the signed-in homepage and the view=teacher page is wrong (it's the nominate image). Can we change it to correct image?

**Review by islemaster — APPROVED** [review]

File: apps/src/templates/ProfessionalLearningApplyBanner.jsx [hunk 2]
```diff
@@ -179,7 +179,7 @@ class ProfessionalLearningApplyBanner extends React.Component {
               >
                 <div>
                   <button type="button" style={this.styles.button}>
-                    Apply Now
+                    Join us
```
Comment (Hamms) [inline apps/src/templates/ProfessionalLearningApplyBanner.jsx:182]: can this be translated?

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 3]
```diff
@@ -134,7 +142,7 @@
     .right.col-50.mobile-center{style: "margin-top: 14px; padding-left: 5px; padding-right: 5px; text-align: center"}
       %div
         %img{src: "images/homepage/codebreak_logo_hand.png", style: "width: 400px; max-width: 100%"}
-      %h3{style: "font-weight: 800; color:#191919; font-size: 16px;"}Take a Code Break! Your weekly dose of inspiration, community, and computer science.
+      %h3{style: "font-weight: 800; color:#191919; font-size: 16px;"}Take a Code Break! Your weekly dose of inspiration, community, and computer science. And check out our other home learning resources!
```
Comment (Hamms) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml:145]: also this

File: apps/src/templates/ProfessionalLearningApplyBanner.jsx (hunk 2 above)
Comment (breville) [reply apps/src/templates/ProfessionalLearningApplyBanner.jsx:182]: Professional Learning is only done in en...

File: pegasus/sites.v3/code.org/views/homepage_hero.haml (hunk 3 above)
Comment (breville) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml:145]: Code Break content is also only done in en...

File: apps/src/templates/ProfessionalLearningApplyBanner.jsx (hunk 2 above)
Comment (Hamms) [reply apps/src/templates/ProfessionalLearningApplyBanner.jsx:182]: oh, looking at the code it looked like we were displaying these to everyone, regardless of language

Comment (breville) [reply apps/src/templates/ProfessionalLearningApplyBanner.jsx:182]: It's true we don't do a language check, but `ProfessionalLearningApplyBanner` is used by `YourSchool` which, while partially translated (to my surprise), is only intended for US schools.

**Pull request merged**